test/components/embedders/test_sentence_transformers_text_embedder.py

```diff
@@ -255,3 +266,25 @@ def test_run_trunc(self):
 
         assert len(embedding_def) == 768
         assert len(embedding_trunc) == 128
+
+    @pytest.mark.integration
+    def test_run_quant(self):
+        """
+        sentence-transformers/paraphrase-albert-small-v2 maps sentences & paragraphs to a 768 dimensional dense vector space
+        """
+        checkpoint = "sentence-transformers/paraphrase-albert-small-v2"
+        text = "a nice text to embed"
+
+        embedder_def = SentenceTransformersTextEmbedder(model=checkpoint, precision="int8")
+        embedder_def.warm_up()
+        result_def = embedder_def.run(text=text)
+        embedding_def = result_def["embedding"]
+
+        check = 0
+
+        for emb in embedding_def:
+            if not isinstance(emb, int):
+                check += 1
+
+        assert len(embedding_def) == 768
+        assert check == 0
```

Comment:
```suggestion
        assert len(embedding_def) == 768
        assert(all(isinstance(el,int) for el in embedding_def))
```

small suggestion :-)